Pcap++/src/DpdkDevice.cpp

```diff
@@ -841,9 +839,7 @@ uint16_t DpdkDevice::receivePackets(MBufRawPacket** rawPacketsArr, uint16_t rawP
 	uint16_t packetsReceived = rte_eth_rx_burst(m_Id, rxQueueId, mBufArray, rawPacketArrLength);
 
 	if (unlikely(packetsReceived <= 0))
-	{

```

Comment:
It's a better practice to add curly brackets `{...}` even when the `if` clause includes one statement - it's more readable that way.

Same for the other changes in this file and in `Pcap++/src/KniDevice.cpp`

Reply:
Also while revoking i forgot to fix unsigned less than zero warning. Fixed now.